### PR TITLE
chore: refactor TemplateSizeLimit integration test

### PR DESCRIPTION
**Pull request by u2**

### CODE REVIEW

File: test/src/specs/mining/size_limit.rs [hunk 1]
```diff
@@ -11,14 +12,24 @@ impl Spec for TemplateSizeLimit {
         let node = &net.nodes[0];
 
         info!("Generate 1 block");
-        node.generate_block();
+        let blank_block = node.new_block(None, None, None);
+        let _ = node.submit_block(&blank_block.data());
```
[the remark this answers is not in the document]
Comment (u2) [reply test/src/specs/mining/size_limit.rs]: thanks

[the remark this answers is not in the document]
Comment (u2) [reply test/src/specs/mining/size_limit.rs]:
```
    pub fn submit_block(&self, block: &Block) -> Byte32 {
        self.rpc_client()
            .submit_block("".to_owned(), block.clone().into())
            .expect("submit_block result none")
    }
```

[the remark this answers is not in the document]
Comment (jjyr) [reply test/src/specs/mining/size_limit.rs]:
```suggestion
        node.submit_block(&blank_block.data());
```

File: test/src/specs/mining/size_limit.rs [hunk 2]
```diff
@@ -28,14 +36,19 @@ impl Spec for TemplateSizeLimit {
             txs_hash.push(hash.clone());
         });
 
+        // skip proposal window
+        let _ = node.generate_block();
```
Comment (jjyr) [inline test/src/specs/mining/size_limit.rs]:
```suggestion
        node.generate_block();
```

**Comment by u2** [comment]
> Please remove unnessacery `let _ =`, it's not best practice, and may introduce error if return value change to Result

Thanks.

**Comment by u2** [comment]
bors r=jjy,quake

**Pull request merged**